MediaBrowser.Model/Configuration/ServerConfiguration.cs

```diff
@@ -271,7 +271,7 @@ public ServerConfiguration()
 
             SortReplaceCharacters = new[] { ".", "+", "%" };
             SortRemoveCharacters = new[] { ",", "&", "-", "{", "}", "'" };
-            SortRemoveWords = new[] { "the", "a", "an" };
+            SortRemoveWords = new[] { "the", "a", "an", "der", "die", "das" };
```

Comment:
Could this inadvertently affect English titles since 'die' is also English? For example, I think this would cause the movie "Die Hard" to get sorted under 'H'?

Reply:
Oh, yes, you are completely right, i didn't think anything of it.
But do you know another title which would be affected? On the other hand, there are a way more titles in german which would be sorted correctly.
And what would be the right way to solve this and what if i rename the files to something like "Herr der Ringe, Der" resp. "Lord of the Rings, The"?

Reply:
We have lots of fields here that aren't exposed in the dashboard, which makes no sense when they're part of the configuration. A better solution might be defaulting to English and simply exposing this option (and hopefully all of them eventually) in the API for server settings.

Reply:
Alternatively, this could go into a translation resource file so we can have a separate word list for each language. However, that would only be useful if we know the title language used for each item, but I don't know if that information is saved.

Reply:
This configuration object is already accessible for modification from the UI, we can just add an option to allow editing it.
I highly doubt we would be able to compose a proper list for all the languages we expect to support...

However, making it detect the title language somehow and apply correct patterns would be even better in the long run.

Reply:
A proper (predefined) list which works in all languages and with individual collections would not be easy, that's right. I think that it should be a user's choice (like in MythTV), because each collection is different. To expose the list is a great idea, thanks @dkanada !